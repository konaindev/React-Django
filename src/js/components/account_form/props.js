export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
