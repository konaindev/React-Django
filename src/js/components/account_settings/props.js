const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
