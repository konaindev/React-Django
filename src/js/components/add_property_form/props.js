export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
