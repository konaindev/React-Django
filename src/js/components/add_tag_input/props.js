const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
