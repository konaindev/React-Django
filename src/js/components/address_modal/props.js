export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
