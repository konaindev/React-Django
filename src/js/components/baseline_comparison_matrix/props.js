import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
