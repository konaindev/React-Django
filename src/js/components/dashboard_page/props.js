export const props = {
  properties: [
    {
      property_name: "Meridian SLC",
      property_id: "pro_1prk0l2sdc0jv8dr",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 2,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Beridian SLC",
      property_id: "pro_ojso5xourpuai2vs",
      address: "",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 1,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Teridian SLC",
      property_id: "pro_fy9oyksyn2cbvop2",
      address: "1779 Nacogdoches Rd, San Antonio, TX 78209, USA",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 2,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Deridian SLC",
      property_id: "pro_5p1zpema2mezfyu0",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 0,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Peridian SLC",
      property_id: "pro_1aww0vi1utykrzt3",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 2,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Leridian SLC",
      property_id: "pro_gmcmkdnapem2pwvc",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 2,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Qeridian SLC",
      property_id: "pro_r3izthx0760jgqh0",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 1,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Keridian SLC",
      property_id: "pro_i6uvh7d5lk0ux3sf",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 1,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Yeridian SLC",
      property_id: "pro_xr068tps5ero17ur",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 2,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    },
    {
      property_name: "Weridian SLC",
      property_id: "pro_uif27s7rpx8ewwx5",
      address: "123 4th Avenue, Seattle WA 98106",
      image_url: "https://i.imgur.com/UEH4gfU.jpg",
      performance_rating: 2,
      url: "https://app.remarkably.io/projects/pro_nqcu73oiinomuvn7/baseline/"
    }
  ],
  user: {
    email: "test@remarkably.io",
    user_id: "1234556960",
    account_id: "5678899",
    account_name: "Remarkably Client",
    profile_image_url:
      "https://storage.googleapis.com/tracker-avatar-production/a11569ffecab2b5564c0cbfa5e5f283a_3140620_96.png",
    logout_url: "/user/logout",
    account_url: "/user/1234556960"
  },
  search_url: "/dashboard",
  locations: [
    { city: "Seattle", label: "Seattle, WA", state: "wa" },
    { city: "New Haven", label: "New Haven, CT", state: "ct" },
    { city: "Dallas", label: "Dallas, TX", state: "tx" }
  ],
  property_managers: [
    { id: "12345", label: "Alliance Property Management" },
    { id: "45678", label: "Red Stone Property Management" }
  ],
  asset_managers: [
    { id: "12345", label: "Brickhouse Asset Management" },
    { id: "54321", label: "Brick and Mortar Capital" }
  ],
  funds: [
    { id: "12345", label: "Ground Up Development Fund 1" },
    { id: "54321", label: "Underwater Asset Fund 5" }
  ]
};
