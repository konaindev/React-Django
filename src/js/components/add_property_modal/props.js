import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
