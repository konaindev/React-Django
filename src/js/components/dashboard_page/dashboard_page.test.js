import renderer from "react-test-renderer";
import { Provider } from "react-redux";

import storeFunc from "../../state/store";

import DashboardPage from "./index";
import { props } from "./props";

const { store } = storeFunc();

describe("DashboardPage", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <DashboardPage {...props} />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("Row view", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <DashboardPage {...props} viewType="row" />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
  it("Row select", () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <DashboardPage
            {...props}
            viewType="row"
            selectedProperties={props.properties.slice(0, 1)}
          />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
