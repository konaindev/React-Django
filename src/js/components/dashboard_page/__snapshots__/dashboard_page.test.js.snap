// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardPage Row select 1`] = `
<div
  className="chrome"
>
  <div
    className="page-chrome__top-chrome"
  >
    <div
      className="page-chrome__top-chrome__content"
    >
      <div
        className="page-header"
      >
        <div
          className="container page-header__inner"
        >
          <a
            href="/dashboard"
          >
            <div
              className="remarkably-logo"
            >
               
            </div>
          </a>
        </div>
      </div>
    </div>
    <div
      className="page-chrome__top-chrome__spacing-fix"
      style={
        Object {
          "height": 76,
        }
      }
    >
       
    </div>
  </div>
  <div
    className="dashboard-content"
  >
    <p>
      Please contact your Account Manager to setup access to your properties
    </p>
  </div>
</div>
`;

exports[`DashboardPage Row view 1`] = `
<div
  className="chrome"
>
  <div
    className="page-chrome__top-chrome"
  >
    <div
      className="page-chrome__top-chrome__content"
    >
      <div
        className="page-header"
      >
        <div
          className="container page-header__inner"
        >
          <a
            href="/dashboard"
          >
            <div
              className="remarkably-logo"
            >
               
            </div>
          </a>
        </div>
      </div>
    </div>
    <div
      className="page-chrome__top-chrome__spacing-fix"
      style={
        Object {
          "height": 76,
        }
      }
    >
       
    </div>
  </div>
  <div
    className="dashboard-content"
  >
    <p>
      Please contact your Account Manager to setup access to your properties
    </p>
  </div>
</div>
`;

exports[`DashboardPage renders correctly 1`] = `
<div
  className="chrome"
>
  <div
    className="page-chrome__top-chrome"
  >
    <div
      className="page-chrome__top-chrome__content"
    >
      <div
        className="page-header"
      >
        <div
          className="container page-header__inner"
        >
          <a
            href="/dashboard"
          >
            <div
              className="remarkably-logo"
            >
               
            </div>
          </a>
        </div>
      </div>
    </div>
    <div
      className="page-chrome__top-chrome__spacing-fix"
      style={
        Object {
          "height": 76,
        }
      }
    >
       
    </div>
  </div>
  <div
    className="dashboard-content"
  >
    <p>
      Please contact your Account Manager to setup access to your properties
    </p>
  </div>
</div>
`;
